src/server/editorServices.ts

```diff
@@ -3632,6 +3640,7 @@ namespace ts.server {
             if (proj.typeAcquisition.enable === undefined) {
                 proj.typeAcquisition.enable = hasNoTypeScriptSource(proj.rootFiles.map(f => f.fileName));
             }
+            proj.typeAcquisition.inferTypings = this.hostConfiguration.typeAcquisition?.inferTypings ?? true;

```

Comment:
I'm a little surprised by this - it seems like whatever the editor sends on the specific project is unconditionally overwritten by the host config. Is that intended?

Reply:
That was my intent. Do you think we should sent the setting with `openExternalProjects` rather than making it configurable through `configure`? 

I guess I don't understand why we would prefer that way.

Reply:
Yes.. This should be part of open external projects